externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/events/EngineEventsStore.scala

```diff
@@ -61,6 +61,14 @@ class EngineEventsStore(store: KVStore) {
     }
   }
 
+  def getStatementLineage(executionId: String): Option[SparkOperationLineageEvent] = {
+    try {
```

Comment:
This method has never been called?

Reply:
> This method has never been called?

Yes, it's just for convenience we use it later such as displaying lineage in the Spark UI. do i need to remove it?